recipes/git-python/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "git-python" %}
+{% set version = "1.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/pynickle/git-python
```

Comment:
Please use a release with a hash that we can freeze in time and ensure the build will be reproducible.

Reply:
added `git_rev` with the git commit hash, which freezes in time